l3_msgs/Foothold foot
std_msgs/String target_frame
---
l3_msgs/Foothold foot
l3_footstep_planning_msgs/ErrorStatus status
